### PR TITLE
[Docs] Add npm init to install proceedure

### PR DESCRIPTION
**Pull request by joshin-run**
I'm learning TAPE and thought it would have been helpful to have some of the info that I added. I'm looking for a dev job and I wanted to add this information in for the community. Looks like a great test library. Thanks!

### CODE REVIEW

File: readme.markdown [hunk 1]
```diff
@@ -374,11 +374,24 @@ $ node object.js test/x.js test/y.js
 
 # install
 
-With [npm](https://npmjs.org) do:
+With [npm](https://npmjs.org) proceed through the following steps:
 
+- run npm init and set up your package.json: this tells you what node modules others need when working on your repo
```
Comment (ljharb) [inline readme.markdown]: hmm - this is kind of required knowledge to be able to use any npm package; i'm not sure how valuable it is here.

File: readme.markdown [hunk 2]
```diff
@@ -374,11 +374,24 @@ $ node object.js test/x.js test/y.js
...
+```
+npm init
+```
+- install
 ```
 npm install tape --save-dev
 ```
+- go to [gitignore.io](https://www.gitignore.io/) and type __NODE__
```
Comment (ljharb) [inline readme.markdown]: this definitely isn't necessary, as git knowledge is also a prerequisite for doing development in general - and `jspm_packages` only applies if you're using JSPM.

File: readme.markdown [hunk 3]
```diff
@@ -374,11 +374,24 @@ $ node object.js test/x.js test/y.js
...
+- copy all text and paste into your _.gitignore_ file
+- most relevent to be _ignored_ would be:
```
Comment (ljharb) [inline readme.markdown]: `s/relevent/relevant`

File: readme.markdown (hunk 1 above)
Comment (gabrielcsapo) [reply readme.markdown]: I agree with this

Comment (gabrielcsapo) [reply readme.markdown]: People who are new to node or NPM can visit this https://docs.npmjs.com/getting-started/what-is-npm which incredible useful for getting started.

**Comment by ljharb** [comment]
I think this can be closed.

**Pull request closed without merging**